main.go

```diff
@@ -96,6 +99,25 @@ func showVendored(src *backvendor.GoSource) {
 	}
 }
 
+func readExcludeFile() []string {
+	if *excludeFrom == "" {
+		return nil
+	}
+
+	e, err := os.Open(*excludeFrom)
+	if err != nil {
+		return nil
```

Comment:
Shouldn't the user be told if he specifies an exclude file yet it can't be read, e.g. permissions?

Reply:
Quite right, updated.